website/content/docs/platform/k8s/helm/enterprise.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Running Vault Enterprise - Kubernetes
+description: >-
+  Vault Helm supports deploying Vault Enterprise, including license autoloading.
+---
+
+# Vault Enterprise
+
+You can use this Helm chart to deploy Vault Enterprise by following a few extra steps around licensing.
+
+~> **Note:** As of Vault Enterprise 1.8, the license must be specified via HCL configuration or environment variables on startup, unless the Vault cluster was created with an older Vault version and the license was stored, or the Vault Enterprise binary has the license baked in. More information is available in the [Vault Enterprise License docs](/docs/enterprise/license).
+
+## Vault Enterprise 1.8+
+
+First create a Kubernetes secret using the contents of your license file. For example, the following commands create a secret with the name `vault-ent-license` and key `license`:
+
+```bash
+secret=$(cat 1931d1f4-bdfd-6881-f3f5-19349374841f.hclic)
+kubectl create secret generic vault-ent-license --from-literal="license=${secret}"
+```
+
+-> **Note:** If you cannot find your `.hclic` file, please contact your sales team or Technical Account Manager.
+
+In your chart overrides, set the values of `server.image` to one of the enterprise [release tags](https://hub.docker.com/r/hashicorp/vault-enterprise/tags). Also set the name of the secret you just created in `server.enterpriseLicense`.
+
+```yaml
+# config.yaml
+server:
+  image:
+    repository: hashicorp/vault-enterprise
+    tag: 1.8.0_ent
+  enterpriseLicense:
+    secretName: vault-ent-license
+```
+
+Now run `helm install`:
+
+```shell-session
+$ helm install hashicorp hashicorp/vault -f config.yaml
+```
+
+Once the cluster is [initialized and unsealed](/docs/platform/k8s/helm/run), you may check the license status using the `vault license get` command:
+
+```shell
+kubectl exec -ti vault-0 -- vault license get
+```
+
+## Vault Enterprise prior to 1.8
+
+In your chart overrides, set the values of `server.image` to one of the enterprise [release tags](https://hub.docker.com/r/hashicorp/vault-enterprise/tags). Install the chart, and initialize and unseal vault as described in [Running Vault](/docs/platform/k8s/helm/run).
+
+After Vault has been initialized and unsealed, setup a port-forward tunnel to the Vault Enterprise cluster:
+
+```shell
+kubectl port-forward vault-0 8200:8200
+```
+
+Next, in a separate terminal, create a `payload.json` file that contains the license key like this example:
```

Comment:
I'm ok with it, but out of curiosity, why are we giving instructions using `curl` instead of `vault`?

Reply:
Oh nvm, I see this is just moved.